src/builder/edit_member.rs

```diff
@@ -87,4 +90,40 @@ impl EditMember {
 
         self
     }
+
+    /// Times the user out until `time`, an ISO8601-formatted datetime string.
+    ///
+    /// Requires the [Moderate Members] permission.
+    ///
+    /// [Moderate Members]: crate::model::permission::Permissions::MODERATE_MEMBERS
+    pub fn communication_disabled_until<Tz>(&mut self, time: String) -> &mut Self
+    {
+        self.0.insert("communication_disabled_until", Value::String(time));
+        self
+    }
+
+    /// Times the user out until `time`.
+    ///
+    /// Requires the [Moderate Members] permission.
+    ///
+    /// [Moderate Members]: crate::model::permission::Permissions::MODERATE_MEMBERS
+    pub fn communication_disabled_until_datetime<Tz>(&mut self, time: DateTime<Tz>) -> &mut Self
+    where
+        Tz: TimeZone,
+        Tz::Offset: Display,
+    {
+        let timestamp = time.to_rfc3339();
+        self.0.insert("communication_disabled_until", Value::String(timestamp));
+        self
+    }
+
+    /// Remove a user's timeout, if there is one.
+    ///
+    /// Requires the [Moderate Members] permission.
+    ///
+    /// [Moderate Members]: crate::model::permission::Permissions::MODERATE_MEMBERS
+    pub fn remove_timeout(&mut self) -> &mut Self {
```

Comment:
Seems like a weird name, maybe the other methods should take Option to follow the API or this should be renamed to `enable_communication`

Reply:
I will update this method name.